pom.xml

```diff
@@ -70,6 +70,26 @@
 
         </dependencies>
     </dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```

Comment:
Should be switched to *org.jetbrains.teamcity:tests-support* & TestNG (JUnit execution not binded on Maven build).